proposal/more-math-functions.md

```diff
@@ -0,0 +1,445 @@
+# Math Module
+
+This built-in module is available from the URL `sass:math`.
+
+## Table of Contents
+* [Variables](#variables)
+  * [`e`](#e)
+  * [`pi`](#pi)
+* [Functions](#functions)
+  * [Approximation](#approximation)
+    * [`ceil()`](#ceil)
+    * [`floor()`](#floor)
+    * [`round()`](#round)
+  * [Comparison](#comparison)
+    * [`clamp()`](#clamp)
+    * [`max()`](#max)
+    * [`min()`](#min)
+    * [`percentage()`](#percentage)
+  * [Division](#division)
+    * [`div()`](#div)
+    * [`mod()`](#mod)
+  * [Exponentiation](#exponentiation)
+    * [`log()`](#log)
+    * [`pow()`](#pow)
+    * [`sqrt()`](#sqrt)
+  * [Trigonometry](#trigonometry)
+    * [Procedures](#procedures)
+      * [Verify Inputs](#verify-inputs)
+    * [`cos()`](#cos)
+    * [`sin()`](#sin)
+    * [`tan()`](#tan)
+    * [`acos()`](#acos)
+    * [`asin()`](#asin)
+    * [`atan()`](#atan)
+    * [`atan2()`](#atan2)
+      * [Edge cases](#edge-cases)
+  * [Units](#units)
+    * [`compatible()`](#compatible)
+    * [`is-unitless()`](#is-unitless)
+    * [`unit()`](#unit)
+  * [Util](#util)
+    * [`abs()`](#abs)
+    * [`hypot()`](#hypot)
+    * [`random()`](#random)
+
+## Variables
+
+### `e`
+
+```
+$e
+```
+
+### `pi`
+
+```
+$pi
+```
+
+## Functions
+
+### Approximation
+
+#### `ceil()`
+
+```
+ceil($number)
+```
+This function is also available as a global function named `ceil()`.
+
+#### `floor()`
+
+```
+floor($number)
+```
+
+This function is also available as a global function named `floor()`.
+
+#### `round()`
+
+```
+round($number)
+```
+
+This function is also available as a global function named `round()`.
+
+### Comparison
+
+#### `clamp()`
+
+```
+clamp($min, $number, $max)
+```
+
+* Return `$min` if `$min` >= `$max`
+* Return `$min` if `$number` <= `$min`.
+* Return `$max` if `$number` => `$max`.
+* Return `$number`.
+
+#### `max()`
+
+```
+max($numbers...)
+```
+
+This function is also available as a global function named `max()`.
+
+#### `min()`
+
+```
+min($numbers...)
+```
+
+This function is also available as a global function named `min()`.
+
+#### `percentage()`
+
+```
+percentage($number)
+```
+
+This function is also available as a global function named `percentage()`.
+
+### Division
+
+#### `div()`
+
+```
+div($dividend, $divisor)
+```
+
+This function is also available as a global function named `divide()`.
+
+* Throw error if the units of `$dividend` and `$divisor` are not
+  [compatible](#compatible).
+* Return `$dividend` / `$divisor`.
```

Comment:
I think you can leave this out, since it's already covered by [the slash-separator proposal](https://github.com/sass/sass/blob/master/accepted/slash-separator.md#dividemathdiv-function).

Reply:
Removed.